config/application.rb

```diff
@@ -11,12 +13,10 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
 
-    config.active_job.queue_adapter = :delayed_job
-
-    config.time_zone = 'Madrid'
```

Comment:
we must be aware of the meetings times before and after deployment. And configure adequately at organization leve